simapp/app.go

```diff
@@ -165,7 +162,7 @@ type SimApp struct {
 	SlashingKeeper   slashingkeeper.Keeper
 	MintKeeper       mintkeeper.Keeper
 	DistrKeeper      distrkeeper.Keeper
-	GovKeeper        govkeeper.Keeper
+	GovKeeper        *govkeeper.Keeper
```

Comment:
A pointer to `GovKeeper` is require since we're setting hooks and routes in a post-construction step.